README.md

```diff
@@ -1,210 +1,171 @@
-    "vega": "^5.7.3",
+    "styled-system": "^5.1.5",
+    "vega": "^5.17.3",
     "vega-embed": "^6.0.0",
-    "vega-lite": "^4.0.0-beta.10"
+    "vega-lite": "^5.0.0",
+    "vega-tooltip": "^0.27.0"
+```
+
+- Install the dependencies :
+
+```sh
+npm install
 ```
 
-- Run ```npm install``` to install dependencies
+### Usage
+
+- Import a component from ```@scality/core-ui/dist/next'``` or ```@scality/core-ui```
+
+- Use props to change its appearance and behaviour
+
+```jsx
+import { Button } from '@scality/core-ui/dist/next';
+import { Icon } from '@scality/core-ui';
+
+<Button variant="primary" onClick={handleClick} label="Save" icon={<Icon name="Save" />} />
+
+```
+
+To learn more about the available components, you can read the [documentation](https://scality.github.io/core-ui/)
+
+### Theming
+
+Components are themable by using the [styled-components theming concept](https://www.styled-components.com/docs/advanced). \
+Wrap your app in a ```ThemeProvider``` and provide it a theme.
+The theme needs to be defined as below :
 
-- Import a component (e.g. Layout). You can wrap your component in a styled-components'```ThemeProvider``` to override its [default theme](https://github.com/scality/core-ui/blob/6f5a7946e7086e08883a8fe48182598ce8a476e5/src/lib/style/theme.js#L34) by defining your own theme like below.
-```javascript
+```jsx
 
 import { ThemeProvider } from 'styled-components';
 import { Layout } from '@scality/core-ui';
-import '@fortawesome/fontawesome-free/css/all.css';
-
-const themes = [
-  {
-    name: "Light Theme",
-    brand: {
-      alert: "#A39300",
-      base: "#607080",
-      primary: "#FAF9FB",
-      primaryDark1: "#F7F6F9",
-      primaryDark2: "#EDEAF0",
-      secondary: "#037AFF",
-      secondaryDark1: "#1C3D59",
-      secondaryDark2: "#1C2E3F",
-      success: "#006F62",
-      healthy: "#24871D",
-      healthyLight: "#33A919",
-      warning: "#946F00",
-      danger: "#AA1D05",
-      critical: "#BE321F",
-      background: "#ffffff",
-      backgroundBluer: "#ECF4FF",
-      textPrimary: "#313B44",
-      textSecondary: "#8593A0",
-      textTertiary: "#A7B6C3",
-      borderLight: "#EBEBEB",
-      border: "#A5A5A5",
-      info: "#8C8C8C"
-    }
+
+const theme = {
+    statusHealthy: '#0AADA6',
+    statusHealthyRGB: '10,173,166',
+    statusWarning: '#F8F32B',
+    statusWarningRGB: '248,243,43',
+    statusCritical: '#E84855',
+    statusCriticalRGB: '232,72,85',
+    selectedActive: '#037AFF',
+    highlight: '#1A3C75',
+    border: '#4A4A4A',
+    buttonPrimary: '#2F4185',
+    buttonSecondary: '#595A78',
+    buttonDelete: '#3D0808',
+    infoPrimary: '#8E8EAC',
+    infoSecondary: '#333366',
+    backgroundLevel1: '#121219',
+    backgroundLevel2: '#323245',
+    backgroundLevel3: '#232331',
+    backgroundLevel4: '#1B1B27',
+    textPrimary: '#EAEAEA',
+    textSecondary: '#B5B5B5',
+    textTertiary: '#DFDFDF',
+    textReverse: '#000000',
+    textLink: '#71AEFF',
   },
```

Comment:
I think we should say that we have 2 defaults themes that they can find here : https://github.com/scality/core-ui/blob/development/1.0/src/lib/style/theme.ts

If you need to create a new one, it's possible but you need to follow the type `CoreUITheme`.